test/parsing-test.js

```diff
@@ -30,7 +30,22 @@ describe('Messages parsing', function () {
         sendGst.restore()
         getRoom.restore()
     })
-
+    it('starting with bot should reply help message', function () {
```

Comment:
No, this is not correct. I would suggest it only replies with help message if the message starts with bot and only contains bot or you have bot help e.g.

```
bot
bot help
/bot
/bot help
```

Anything else should not generate a help message.